app/src/main/java/org/wikipedia/onboarding/OnboardingPageView.kt

```diff
@@ -81,6 +81,10 @@ class OnboardingPageView constructor(context: Context, attrs: AttributeSet? = nu
         }
     }
 
+    fun hideTertiaryTextView(isVisible: Boolean) {
```

Comment:
This function seems misnamed. I would expect `hideTertiaryTextView(true)` to hide the view, not show it. It should probably be something like `setTertiaryTextViewVisible(...)`